globals.json

```diff
@@ -1113,7 +1117,8 @@
 		"UI": false,
 		"Utils": false,
 		"WebApp": false,
-		"WebAppInternals": false
+		"WebAppInternals": false,
+		"Promise": false
```

Comment:
True, but ESLint (in which this module is used) marked `Promise` as undefined, i.e. it didn't pick it up as a whitelisted global. But it makes sense to remove it from the Meteor env, since it's in ES2015 anyway.